docs/source/roadmap.rst

```diff
@@ -64,6 +61,15 @@ Submit a PR!
 Release History
 ---------------
 
+.. rubric:: Done in v3.3
+
+* **RHEL/CentOS:** Drop support for RHEL/CentOS 6.x.
+* **Mistral deprecation:** Orquesta replaces Mistral as the workflow engine.
+* **HipChat Removal:** The HipChat adapter has been removed from st2chatops.
+* **Chef:** Deprecated `chef-stackstorm <https://github.com/stackstorm/chef-stackstorm>`_ deployment.
+* **Docker:** Overhaul for `st2-docker <https://github.com/stackstorm/st2-docker>`_ deployment.
+* **MongoDB:** Require MongoDB 4.0 across all OSes.
```

Comment:
:+1: for adding this to the roadmap!

I'd better reword "Switch to MongoDB 4.0 across all OSes." or similar.